evalml/pipelines/components/transformers/preprocessing/featuretools.py

```diff
@@ -166,4 +166,8 @@ def _handle_partial_dependence_fast_mode(self, pipeline_parameters, X, target):
                 raise ValueError(
                     "Cannot use fast mode with DFS Transformer when features are unspecified or not all present in X.",
                 )
+            # Pass in empty list of features so we don't run calculate feature matrix
+            # which would happen with the full set of features for a single column at refit
+            dfs_transformer["features"] = []
+            pipeline_parameters["DFS Transformer"] = dfs_transformer
```

Comment:
A clarification for my own understanding:  we only use this function with the single column pipeline, correct? This code would never run when we're generating the transformations from the full dataset?

Reply:
Correct - the parameters returned here will only be used to train a pipeline on a single column, and the check above this that confirms all features have corresponding columns in X confirms that calculate feature matrix is never called in the original pipeline; this parameter change makes that the case for the single column as well.

Reply:
yep totally doable